zerver/lib/import_realm.py

```diff
@@ -598,6 +598,19 @@ def bulk_import_client(data: TableData, model: Any, table: TableName) -> None:
             client = Client.objects.create(name=item['name'])
         update_id_map(table='client', old_id=item['id'], new_id=client.id)
 
+
+def fix_subscriptions_is_user_active_column(data: TableData) -> None:
+    """
+    This function is to be called only after UserProfile table has been imported
+    into the database so that queries can be made.
+    """
```

Comment:
Are we sure we want to do this transformation after populating the database?  I can see an argument for just doing everything locally, in memory, before we insert into Subscriber.

Reply:
@mateuszmandera pointed out to me on chat that we're actually doing the Subscriber stuff in memory, which is the important thing to me.  There are probably opportunities to minimize round trips back to the DB for UserProfile (i.e. only do it one time, not once for every table that needs stuff), but it's not a huge concern.

Reply:
I think we actually have the `UserProfile` objects available locally.